assets/src/components/AssignmentGoalInput.js

```diff
@@ -32,7 +32,7 @@ function AssignmentGoalInput (props) {
   } = props
 
   const [goalGradeInternal, setGoalGradeInternal] = useState(goalGrade)
-  const debouncedGoalGrade = useRef(debounce(q => setGoalGrade(q), 500)).current
+  const debouncedGoalGrade = useRef(debounce(q => setGoalGrade(q), 250)).current
```

Comment:
Why was the number reduced?

Reply:
When I added a long debounce to the snackbar, the duration of this save would stack on top of the duration for the  snackbar pop up. It felt quite long compared to other inputs on the page, so I reduced this debounce amount. I didn't consider if there was any drawback to lowering this, other than the visualizations and input amounts rendering slightly faster

Reply:
If it improve overall objective of the view that it is fine